cmds/ocm/commands/ocmcmds/common/addhdlrs/options.go

```diff
@@ -8,30 +8,51 @@ import (
 )
 
 type Options struct {
-	Replace bool
+	Replace           bool
+	PreserveSignature bool
```

Comment:
These Options need some docs I think for incode referencing

Reply:
done